warehouse/integrations/testhelper/setup.go

```diff
@@ -242,3 +250,166 @@ func EnhanceWithDefaultEnvs(t testing.TB) {
 		t.Setenv("LOG_LEVEL", "DEBUG")
 	}
 }
+
+func UploadLoadFile(
+	t testing.TB,
+	fm filemanager.FileManager,
+	fileName string,
+	tableName string,
+) filemanager.UploadedFile {
+	t.Helper()
+
+	f, err := os.Open(fileName)
+	require.NoError(t, err)
+	defer func() { _ = f.Close() }()
+
+	loadObjectFolder := "rudder-warehouse-load-objects"
+	sourceID := "test_source-id"
+
+	uploadOutput, err := fm.Upload(
+		context.Background(), f, loadObjectFolder,
+		tableName, sourceID, uuid.New().String()+"-"+tableName,
+	)
+	require.NoError(t, err)
+
+	return uploadOutput
+}
+
+func RecordsFromWarehouse(
+	t testing.TB,
+	db *sql.DB,
+	query string,
+) [][]string {
+	t.Helper()
+
+	rows, err := db.QueryContext(context.Background(), query)
+	require.NoError(t, err)
+	defer func() { _ = rows.Close() }()
+
+	_ = rows.Err()
+
+	columns, err := rows.Columns()
+	require.NoError(t, err)
+
+	var records [][]string
+	for rows.Next() {
+		resultSet := make([]any, len(columns))
+		resultSetPtrs := make([]any, len(columns))
+		for i := 0; i < len(columns); i++ {
+			resultSetPtrs[i] = &resultSet[i]
+		}
+
+		require.NoError(t, rows.Scan(resultSetPtrs...))
+
+		records = append(records, lo.Map(resultSet, func(item any, index int) string {
+			switch item := item.(type) {
+			case time.Time:
+				return item.Format(time.RFC3339)
+			default:
+				return cast.ToString(item)
+			}
+		}))
+	}
+	return records
+}
+
+func LoadRecords() [][]string {
```

Comment:
Is this loading any records though? It's just returning a matrix of slices.

Reply:
Same comment goes for the other functions.

Reply:
These are helpers for the data we loaded in different scenarios. Update the function name and the added comment as well.